packages/rancher-windows-gmsa/charts/templates/credentialspec.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if .Values.credential.enabled -}}
+apiVersion: windows.k8s.io/v1
+kind: GMSACredentialSpec
+metadata:
+  name: {{ lower .Values.credential.domainJoinConfig.machineAccountName }}  #This is an arbitrary name but it will be used as a reference
+  labels:
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+credspec:
+  ActiveDirectoryConfig:
+    GroupManagedServiceAccounts:
+      - Name: {{ .Values.credential.domainJoinConfig.machineAccountNamename }}   #Username of the GMSA account
+        Scope: {{ .Values.credential.domainJoinConfig.netBiosName }}  #NETBIOS Domain Name
+      - Name: {{ .Values.credential.domainJoinConfig.machineAccountNamename }}   #Username of the GMSA account
+        Scope: {{ .Values.credential.domainJoinConfig.dnsName }} #DNS Domain Name
+  CmsPlugins:
+    - ActiveDirectory

```

Comment:
Should this also be parametrized?

Reply:
I don't believe there are currently any other plugins available.